src/endpoint/s3/ops/s3_post_bucket_delete.js

```diff
@@ -22,6 +23,11 @@ async function post_bucket_delete(req) {
         objects
     });
 
+    if (objects.length > 1000) {
+        dbg.error('cannot delete more than 1000 objects');
```

Comment:
Did you check how AWS writes the message?

Reply:
aws simply returns the error for malformed xml, I added this line so it will be easier to understand what went wrong